### ISSUE TITLE
Convert to v2 addon

### ISSUE THREAD

**Issue by SergeAstapov** [tracked-tools/tracked-toolbox#60]
As this addon provides only run-time functionality hence it's the great candidate for v2 addon conversion.

My thought process is following:

1. [x] land #57, #58, #59
1. [ ] release new minor 1.3.0
1. [x] land #44
1. [ ] convert to monorepo #63
1. [ ] convert addon to v2
1. [ ] release new major 2.0.0

@chriskrycho any thoughts, concerns?

**Comment by chriskrycho** [comment]
On this one, that seems good to me here, with the caveat that we need to cut a minor release bumping this to use the latest major of tracked-maps-and-sets first—that unblocks some consumers who are unable to use the current major of this at the same time as they are using the current major of tracked-built-ins, which is depending on a newer version of tracked-maps-and-sets.

(Related: if you want to write an RFC, I can mentor writing one to get the whole tracked-built-ins stack folded into a single package that we ship with the default blueprint etc.: it's very much needed and we discussed doing exactly that at a recent Framework Core meeting.)

**Comment by SergeAstapov** [comment]
@chriskrycho now I got what you meant. However this addon does not depend on `tracked-maps-and-sets` any more.

Which means you may cut minor right now and I'll start working on v2 conversion which indeed will be slated for next major.

**Comment by chriskrycho** [comment]
Ah, nice – I didn't remember that. Very good! I'll cut a minor release tomorrow bundling up all the stuff we've done lately and then we can get a new major out.

**Comment by boris-petrov** [comment]
@chriskrycho - still waiting on this release! :)

**Comment by SergeAstapov** [comment]
Created https://github.com/tracked-tools/tracked-toolbox/issues/65 instead of this one as v2 addon format is just part of the major release.
